src/main/java/attendance/domain/MenuCommand.java

```diff
@@ -0,0 +1,44 @@
+package attendance.domain;
+
+import attendance.controller.OptionCheckAttendance;
+import attendance.controller.OptionLookupAttendance;
+import attendance.controller.OptionLookupExpulsion;
+import attendance.controller.OptionModifyAttendance;
+import java.util.Arrays;
+
+public enum MenuCommand {
+    ATTEND("1", OptionCheckAttendance.class),
+    MODIFY("2", OptionModifyAttendance.class),
+    LOOKUP("3", OptionLookupAttendance.class),
+    EXPEL("4", OptionLookupExpulsion.class),
+    QUIT("Q", null);
+
+    private static final String MENU_COMMAND_ERROR_MESSAGE = "[ERROR] 지원하지 않는 기능 값입니다.";
+    private static final String COMMAND_PATTERN = "^[1-4|Q]$";
+
+    private final String command;
+    private final Class<?> option;
```

Comment:
```suggestion
    private final MenuOption option;
```

1. Class<?> 로 선언하면 어떤 문제점이 있을까요~?
2. MenuOption 타입으로 선언하면 안될까요?

Reply:
음, 기존에 사용했던 1번의 경우 새 객체가 매번 필요하고 클래스를 동적으로 추가해야 하는 경우에 더 적절하다는 생각이 들었어요. 그래서 굳이 Class<?>를 쓰는 것보다는 2번처럼 선언해 MenuOption 인스턴스들을 저장해두도록 변경했습니다!